packages/models-library/src/models_library/api_schemas_dynamic_sidecar/containers.py

```diff
@@ -1,9 +1,10 @@
+from typing import TypeAlias
+
 from pydantic import BaseModel, NonNegativeFloat
 
 
 class InactivityResponse(BaseModel):
     seconds_inactive: NonNegativeFloat | None = None
 
-    @property
-    def is_inactive(self) -> bool:
```

Comment:
I would leave this and avoid `seconds_inactive` to be nullable (default to 0 instead). 

Try to think of this model as some sort of activity info/report instead of focusing so much in the inactivity. More details about this below

Reply:
dropped this